src/components/CasesPage/CasesPage.scss

```
@@ -0,0 +1,239 @@
+@import 'src/styles/mixins/font';
+@import 'src/styles/mixins/font-scale';
+@import 'src/styles/variables';
+@import 'src/styles/mixins/breakpoint';
```

Let's use `@use` approach. Check out how it is done in `SectionCard.scss`
```suggestion
@use 'src/styles/mixins' as m;
@use 'src/styles/variables' as v;
```

---

Fixed